### ISSUE TITLE
Helm Chart

### ISSUE THREAD

**Issue by Rudedog9d** [loomio/loomio-deploy#110]
My use case included deploying Loomio to Kuberenetes. Would this repo be interested in a helm chart if I submitted a PR?

**Comment by robguthrie** [comment]
Yeah, I'd be interested to see it. However, I may not publish it as I'm not prepared to support it.

As Loomio seems to be valuable to your organization, could I please request that you consider becoming a sponsor of the project.
